### PR TITLE
[performance](load) fix broker load scan ranges for unsplittable files

### PR DESCRIPTION
**Pull request by kaijchen**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Check List (For Committer)

- Test 

    - [ ] Regression test
    - [x] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No colde files have been changed.
        - [ ] Other reason 

- Behavior changed:

    - [x] No.
    - [ ] Yes. 

- Does this need documentation?

    - [x] No.
    - [ ] Yes. 

- Release note

    
    
    None

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

File: fe/fe-core/src/test/java/org/apache/doris/datasource/FileGroupIntoTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+
+public class FileGroupIntoTest {
+
+    private static Stream<Arguments> provideParameters() {
+        return Stream.of(
+            // 6, 1+5, 2+4, 3, max=6
```
Comment (liaoxin01) [inline fe/fe-core/src/test/java/org/apache/doris/datasource/FileGroupIntoTest.java:34]:
```suggestion
            // 6, 5, 4+1, 3+2, max=6
```

**Comment by kaijchen** [comment]
run buildall

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**